src/sa_api_v2/admin.py

```diff
@@ -3,6 +3,7 @@
 via django.contrib.admin.
 """
 
+import os
```

Comment:
Are we using this module? I'm not sure I see a place where we are. If not, I'd recommend removing the import.